libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.spec.ts

```diff
@@ -22,7 +21,7 @@ describe('SkyAgGridCellEditorLookupComponent', () => {
   ];
   let fixture: ComponentFixture<SkyAgGridCellEditorLookupComponent>;
   let nativeElement: HTMLElement;
-  let callback: EventCallback | undefined;
+  let callback: (args: Record<string, unknown>) => void | undefined;
```

Comment:
Why are these changes needed for the node version change?

Reply:
They were found during the Node 16 update process. They can technically remain, but we shouldn't be importing from `typedoc` in the libraries' source code.

Reply:
My guess is it was an auto-IDE lookup that caused the import to be added in the first place.